packages/react/src/components/linkbox/Linkbox.module.scss

```diff
@@ -77,16 +77,25 @@
 .headingSmall {
   padding-bottom: 0;
   font-size: var(--fontsize-heading-s);
+  font-weight: 700;
+  letter-spacing: 0.2px;
+  line-height: 1.4;
 }
 
 .headingMedium {
   padding-bottom: 3px;
   font-size: var(--fontsize-heading-m);
+  font-weight: 500;
```

Comment:
This seems same as the heading-m helper class. Is it possible to combine these?

Reply:
- [ ] Use helper classes in linkbox